### PR TITLE
fix: Troubleshooter - Vertical scrolling not working on left panel or calendar 

### LINKED ISSUE #12751: [CAL-2796] Troubleshooter - Vertical scrolling not working on left panel or calendar
When the left panel is higher than the viewport, you are unable to scroll.

In addition to this, there is some weird behaviour causing it to show the content from the bottom up, rather than the top down. As a result, you can't access the back button

This scenario is most likely to occur if you have more than one calendar connected, however it of course depends on the viewport height.

Also, scrolling on the right panel does not appear to be working.

[https://github.com/calcom/cal.com/assets/4536123/48485486-7dc2-47fa-8b4c-ea7e89bb2db7](https://github.com/calcom/cal.com/assets/4536123/48485486-7dc2-47fa-8b4c-ea7e89bb2db7)

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-2796](https://linear.app/calcom/issue/CAL-2796/troubleshooter-vertical-scrolling-not-working-on-left-panel-or)</sub>

### PR DESCRIPTION
**Pull request by mrsourav1**
Fix: #12751
now both left panel and right panel are scrollable seprately....

### CODE REVIEW

File: .vscode/settings.json
Comment (Udit-takkar) [inline .vscode/settings.json:5]: why did you made a change here?

**Comment by Udit-takkar** [comment]
@mrsourav1 do not push yarn.lock file if you are not making any changes in package.json. I have removed the yarn.lock changes

File: .vscode/settings.json
Comment (mrsourav1) [reply .vscode/settings.json:5]: actually i did not its happend automatically i dont know the reason

**Comment by Udit-takkar** [comment]
@mrsourav1 please remove the yarn.lock file. I already removed it before but you again pushed it.

**Review by Udit-takkar — DISMISSED** [review]
LGTM.I'll will wait for @sean-brydon approval before merging

File: .vscode/settings.json
Comment (PeerRich) [inline .vscode/settings.json:5]:
```suggestion
    "source.fixAll.eslint": true
```

**Review by PeerRich — APPROVED** [review]

File: .vscode/settings.json
Comment (Udit-takkar) [reply .vscode/settings.json]: @PeerRich it's actually due to this new vs code update https://stackoverflow.com/questions/77637621/vscode-workspace-settings-change-on-its-own

**Review by sean-brydon — APPROVED** [review]
LGTM also thanks @mrsourav1 and @Udit-takkar

**Comment by sean-brydon** [comment]
For reference: This PR was reverted as it broke scroll in booker

**Pull request merged**
